Chapter2/1-Colors/Window.cs

```diff
@@ -108,27 +108,26 @@ protected override void OnLoad()
             _lightingShader = new Shader("Shaders/shader.vert", "Shaders/lighting.frag");
             _lampShader = new Shader("Shaders/shader.vert", "Shaders/shader.frag");
 
-            // Initialize the vao for the model
-            _vaoModel = GL.GenVertexArray();
-            GL.BindVertexArray(_vaoModel);
-
-            GL.BindBuffer(BufferTarget.ArrayBuffer, _vertexBufferObject);
-
-            var vertexLocation = _lightingShader.GetAttribLocation("aPos");
-            GL.EnableVertexAttribArray(vertexLocation);
-            GL.VertexAttribPointer(vertexLocation, 3, VertexAttribPointerType.Float, false, 3 * sizeof(float), 0);
-
-            // Initialize the vao for the lamp, this is mostly the same as the code for the model cube
-            _vaoLamp = GL.GenVertexArray();
-            GL.BindVertexArray(_vaoLamp);
+            {
```

Comment:
hmm this block indentation is new, isnt it.... is this consistent with the other chapters?

Reply:
Currently this indentation style is consistent with all projects in chapter 2. 
This might make chapter 1 cleaner in the later projects. But I think it's fine for now?

Reply:
This PR provides a chance to make it consistent across all chapters, Im just saying.

Reply:
True, will look into making it more consistent.

Reply:
I think not refactoring the chapter 1 projects make sense as they have another structure when reading. They could absolutely be rewritten into the style of the chapter 2 tutorials (and maybe should), but that is not something I want to do in this PR. That would have to be a later PR.